### PR TITLE
RTL: Extra white bar appears on notice message under RTL

### PR DESCRIPTION
**Pull request by benjaminliugang**
RTL: Extra white bar appears on notice message under RTL

image

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ sudo: required
 dist: trusty
 language: node_js
 node_js:
-  - stable
+  - 10.5.0
```
Comment (benjaminliugang) [inline .travis.yml]: The latest stable version(12.10.0) always make the CI build failed when run with `eval yarn`.

**Comment by benjaminliugang** [comment]
@joshsylvester @elvinfucom @vincentcn @mahesh-svmx, please help review.

**Comment by benjaminliugang** [comment]
@joshsylvester , could you help me merge and release it?

**Comment by joshsylvester** [comment]
@benjaminliugang the version number in the release is wrong - the History.md refers to version 1.3.4, which has already been released. Can you please submit a new PR to fix?

https://github.com/predixdesignsystem/px-alert-message/releases

**Comment by benjaminliugang** [comment]
> @benjaminliugang the version number in this release is wrong - the History.md refers to version 1.3.4, which has already been released. Can you please submit a new PR to fix?
> 
> https://github.com/predixdesignsystem/px-alert-message/releases

Thanks @joshsylvester. Just send another PR for this issue: https://github.com/predixdesignsystem/px-alert-message/pull/51

**Pull request merged**